exercises/grade-school/GradeSchool.example

```diff
@@ -0,0 +1,32 @@
+module GradeSchool (..) where
+
+import Dict exposing (..)
+
+type alias Grade = Int
```

Comment:
Yep. That's interesting... it turns out the elm-format sublime text plugin I'm using doesn't seem to do anything if the file doesn't actually have a `.elm` extension. I revised the grade-school revision commit (now 6b79102) with updated formatting. All of the other examples from this batch seem to be good.